images/k8s-cloud-builder/Dockerfile

```diff
@@ -44,32 +41,60 @@ RUN apt-get -q update \
 # Install gcloud
 # common::set_cloud_binaries() looks for it in this path
 RUN mkdir /opt/google
-RUN curl -sSL https://sdk.cloud.google.com > /tmp/install.sh && \
-    bash /tmp/install.sh --install-dir=/opt/google --disable-prompts
+RUN apt-get -y update \
+    && apt-get -y install \
+        gcc \
+        python2.7 \
+        python-dev \
+        python-setuptools \
+        wget \
+        ca-certificates \
+        # These are necessary for add-apt-respository
+        software-properties-common \
+    # Install Git >2.0.1
+    && apt-get -y update \
+    && apt-get -y install git \
+    # Setup Google Cloud SDK (latest)
+    && curl -sSL https://sdk.cloud.google.com > /tmp/install.sh \
+    && bash /tmp/install.sh --install-dir=/opt/google --disable-prompts \
+    # install crcmod: https://cloud.google.com/storage/docs/gsutil/addlhelp/CRC32CandInstallingcrcmod
+    && pip install -U crcmod \
+    # Clean up
+    && apt-get -y remove \
+        gcc \
+        python-dev \
+        python-setuptools \
+        wget \
+    && rm -rf /var/lib/apt/lists/* \
```

Comment:
My thinking on getting the Google Cloud SDK via apt was around being able to inventory the container contents, which is useful if nothing else for runtime debug, but likely other reasons too.  Being able to docker run and `dpkg -L` or `apt list --installed` and see which gcc, which python-dev, which Google Cloud SDK are in a given binary instance of the container described by this Dockerfile is really important versus "it's whatever was available on the internet at build time".